www_src/components/basic-element/basic-element.jsx

```diff
@@ -40,11 +40,11 @@ var El = React.createClass({
       zIndex: this.props.zIndex,
       touchactive: false
     };
+    this.buttonStyle = initial;
```

Comment:
this was `buttonStyle = this.getInitialState()`, but initialState has dynamic behaviour and it felt like incredibly dangerous and dev-unpredictable code.

Instead the true initial state now gets cached for the purpose that the original code indicated it was for.